### ISSUE TITLE
[Static Site] Breadcrumbs link are not sufficient distinguishable 

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5805]
## Area or Component

Breadcrumb links

## Quick Summary

Ensure links are distinguished from surrounding text in a way that does not rely on color. Link also does not have a proper indicator of the hover/focus state. 

### How to Replicate
Go to getting started page and check the Home link in the breadcrumb navigation.

## Accessibility Violation

Links must be distinguishable without relying on color. https://dequeuniversity.com/rules/axe/4.7/link-in-text-block?application=AxeChrome

## Action Requested

To solve this problem, you need to fix at least (1) of the following:
The link has insufficient color contrast of 2.56:1 with the surrounding text. (Minimum contrast is 3:1, link text: #005ea2, surrounding text: #1b1b1b)
The link has no styling (such as underline) to distinguish it from the surrounding text. 

The hover/focus state of this link also needs to be fixed.

## Screenshots (optional)

![Screenshot 2023-05-15 at 10 18 22 AM](https://github.com/CDCgov/prime-simplereport/assets/103958711/4b07ab52-4440-4395-b051-4eb0b737db33)
